microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md

```diff
@@ -444,7 +444,7 @@ This computer must have direct access to your Microsoft 365 tenant.
    Example: **EdmUploadAgent.exe /UploadData /DataStoreName PatientRecords /DataFile C:\Edm\Hash\PatientRecords.csv /HashLocation C:\Edm\Hash /Schema edm.xml /AllowedBadLinesPercentage 5**
 
    The default format for the sensitive data file is comma-separated values. You can specify a tab-separated file by indicating the "{Tab}" option with the /ColumnSeparator parameter, or you can specify a pipe-separated file by indicating the "|" option.
-   If your sensitive information table has some incorrectly formatted values, but you want to import the remaining data while ignoring invalid rows anyway, you can use the /AllowedBadLinesPercentage parameter in the command. The example above specifies a five percent threshold. This means that the tool will hash and upload the sensitive information table even if up to five percent of the rows are invalid. The default value for this setting is one percent. 
+   If your sensitive information table has some incorrectly formatted values, but you want to import the remaining data while ignoring invalid rows anyway, you can use the /AllowedBadLinesPercentage parameter in the command. The example above specifies a five percent threshold. This means that the tool will hash and upload the sensitive information table even if up to five percent of the rows are invalid. In versions of the tool that support this parameter the default threhsold is zero percent so any erroneous lines will cause an error. 
```

Comment:
```suggestion
   If your sensitive information table has some incorrectly formatted values, but you want to import the remaining data while ignoring invalid rows anyway, you can use the /AllowedBadLinesPercentage parameter in the command. The example above specifies a five percent threshold. This means that the tool will hash and upload the sensitive information table even if up to five percent of the rows are invalid. In versions of the tool that support this parameter, the default threshold is zero percent so any erroneous lines will cause an error. 
```

Reply:
Martin requested that the default value is documented as stated, since the default is different (1%) on the previous versions of the tool where the value is not configurable, and customers may keep using that version for months since it is not auto-updated. So either the way I wrote the line or some other wording that indicates that the default is zero only for the version of the tool where the value is configurable would be required.

Reply:
@Esaggese - Thank you for the explanation. My suggested change, however, is just to correct the typo ("threshold" instead of "threhsold").